cypress/fixtures/sign.json

```diff
@@ -1,3 +1,4 @@
 {
-    "sourceAccount": "GCECUYXE32PPYKPVYOSILACOCUGMEZSDO7GYERC2GKAG2HM34BLMMOMB"
+    "sourceAccount": "GCECUYXE32PPYKPVYOSILACOCUGMEZSDO7GYERC2GKAG2HM34BLMMOMB",
+    "paymentRecipient": "GCECUYXE32PPYKPVYOSILACOCUGMEZSDO7GYERC2GKAG2HM34BLMMOMB"
```

Comment:
Shouldn't the keys be imported from the .env files?

Reply:
That would be the best, then we need to move operation.json too to an env file